src/figuresClasses.ts

```diff
@@ -1,19 +1,85 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+function roundToTwoDecimalPlaces(value: number): number {
+  return Math.floor(value * 100) / 100;
 }
 
-export class Triangle {
-
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
```

Comment:
not necessary because you already declared type in Figure class 
```suggestion
  shape = 'triangle';
```